panels/templates/attractions_admin/checkin.html

```diff
@@ -93,36 +93,11 @@
         $showCheckedInButton = $('#filter'),
         $attendee = $('#attendee');
 
-    $(document).scannerDetection({
-      timeBeforeScanTest: 100,
-      avgTimeByChar: 30,
-      endChar: [9, 13, '\\'],
-      startChar: ['~', '\\'],
-      preventDefault: false,
-      onError: function() {
-        $badgeNum.focus();
-      },
-      onReceive: function(event) {
-        if(event.key === '~' || event.key === '\\') {
-          event.preventDefault();
-          $badgeNum.blur();
-        }
-      },
-      onComplete: function(barcode, qty) {
-        getBadgeNumFromBarcode(barcode, function(response) {
-          $badgeNum.focus();
-          if(response) {
-            if(response['error']) {
-              toastr.error(response['error'], '', {timeOut: 3000});
-            } else if(response['badge_num']) {
-              var badgeNum = response['badge_num'];
-              $badgeNum.val(badgeNum);
-              getSignups(badgeNum);
-            }
-          }
-        });
-      }
-    });
+      {% set detect_barcode_scan = True %}
```

Comment:
Does this actually work? Wouldn't all of the code in `baseextra.html` already be rendered by the time we reach this line (meaning `detect_barcode_scan` would be `False` at the time the barcode plugin `{% extends "baseextra.html" %}`)?

I'm actually not sure! I'm going to pull this branch locally and test it right now!

Reply:
Oh shoot, that's a very good point. I'll try to brainstorm another tactic.

Reply:
Did you ever get this tested?

Reply:
Oh, no! Fell off my radar ☹️